### DISCUSSION TITLE
请问台服是不支持肉鸽吗？

### DISCUSSION THREAD

**Discussion by zxc173123** [MaaAssistantArknights/MaaAssistantArknights#2310] [Q&A]
卡在招募那了

**Comment by MistEO** [comment]
没号 + 没时间适配。有兴趣的大佬们可以试着自己适配下，不需要写代码，但需要大量的截图。最好能再给我们提个 PR（

一些可能参考的链接：
美服肉鸽适配 PR: https://github.com/MaaAssistantArknights/MaaAssistantArknights/pull/1242
日服肉鸽适配 PR: https://github.com/MaaAssistantArknights/MaaAssistantArknights/pull/1233
肉鸽适配讨论及一些小工具的使用方法: https://github.com/MaaAssistantArknights/MaaAssistantArknights/discussions/1212
PR 教程: https://github.com/MaaAssistantArknights/MaaAssistantArknights/blob/master/docs/2.2-开发相关.md

**Reply by MistEO** [reply]
WIP: https://github.com/MaaAssistantArknights/MaaAssistantArknights/pull/2322
